### PR TITLE
security: fix some security vulnerabilities

### PR DESCRIPTION
**Pull request by Bughue**
- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did
存在不少漏洞版本依赖，需要清除替换
https://mvnrepository.com/artifact/io.seata/seata-all/1.6.0

jackson-databind
版本：2.11.4>>2.13.4.1
CVE-2022-42004
CVE-2022-42003

protobuf-java
版本：3.11.4>>3.16.3
CVE-2022-3509
CVE-2022-3171

postgresql
版本：42.1.4>>42.3.3
CVE-2022-26520
CVE-2022-21724

spring-framework
版本：5.3.18>>5.3.20
CVE-2022-22971

nacos(暂未修改)
CVE-2021-43116
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-43116

dubbo
版本：2.6.5>>2.6.10
CVE-2021-30179
CVE-2021-25640

guava
版本：27.0.1-jre>>30.1-jre
CVE-2020-8908

mysql-connector-java
版本：5.1.35>>5.1.42
CVE-2019-2692
CVE-2017-3589
CVE-2017-3523

### Ⅱ. Does this pull request fix one issue?

fixes https://github.com/seata/seata/issues/5171

### CODE REVIEW

File: rm-datasource/pom.xml [hunk 1]
```diff
@@ -67,6 +67,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
+            <version>2.13.4.1</version>
```
Comment (slievrly) [inline rm-datasource/pom.xml:70]: why change version here , not in dependencies.pom ?

File: build/pom.xml [hunk 2]
```diff
@@ -63,7 +63,7 @@
 
     <properties>
         <!-- seata version -->
-        <revision>1.6.1-SNAPSHOT</revision>
+        <revision>1.6.0-SNAPSHOT</revision>
```
Comment (slievrly) [inline build/pom.xml:66]: revert

**Review by a364176773 — APPROVED** [review]
LGTM

File: dependencies/pom.xml [hunk 3]
```diff
@@ -615,6 +615,11 @@
                 <artifactId>jjwt-jackson</artifactId>
                 <version>${jwt.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
```
Comment (slievrly) [inline dependencies/pom.xml:619]: why add this dependency？

Comment (slievrly) [reply dependencies/pom.xml:619]: The version needs to be declared as a variable reference.

**Review by wangliang181230 — APPROVED** [review]
LGTM
change.md加下。

**Comment by Bughue** [comment]
> LGTM change.md加下。

done

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**